scripts/publish-next.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# fail immediately when any script fails
+set -e
+
+# change to project root
+cd $(dirname $0)/..
+
+# Build project
+npm run build
+
+# Go to Cashmere build output directory
+cd $(dirname $0)/../dist/cashmere
+
+# Make it possible to install this version using `npm i -S @healthcatalyst/cashmere@next`
+npm config set tag next
+
+# Set version to timestamped beta version
+npm version 0.0.0-beta.$(date +"%Y%m%d%H%M%S") --git-tag-version false
```

Comment:
I think the proper semver way to do this is 
`npm version prerelease --preid=beta`
https://docs.npmjs.com/cli/version

Reply:
I'm a little fuzzy on how that would work - do we keep the current version number in package.json? I thought that normally the version number is auto-gen'd at release time and not committed to git.
Would it be something like this?
version 6.0.0 is released on `master`, then version 6.0.0.beta is released after on `dev`?  That would be a bit backwards I would think.
I looked at using semantic-release, but that only works with git tags, so we would be tagging every dev release and our master releases would be jumping numbers a lot more, which I don't think we want to do. (also semantic-release doesn't work with a branch named `dev`, we would have to use the name "next" instead)
I think keeping a timestamp in the version number somewhere would be helpful, regardless of how we decide to version these prerelease packages.